src/dict/stardict.cc

```diff
@@ -436,14 +436,15 @@ string StardictDictionary::handleResource( char type, char const * resource, siz
     {
       QString articleText = QString( "<div class=\"sdct_h\">" ) + QString::fromUtf8( resource, size ) + "</div>";
 
-      static QRegularExpression imgRe( R"((<\s*(?:img|script)\s+[^>]*src\s*=\s*["']?)(?!(?:data|https?|ftp):))",
```

Comment:
We should just delete `ftp` because both firefox & chrome stopped ftp support long time ago.

https://chromestatus.com/feature/6246151319715840

Reply:
Or in another PR too remove all `ftp `https://github.com/search?q=repo%3Axiaoyifang%2Fgoldendict-ng%20ftp&type=code

Reply:
this may come from original stardict dictionary.   
If we remove it ,it will become   `bres://dictionaryid/ftp://xxx`

Reply:
we can keep the ftp to keep the program function well ,the chrome does not support it is another issue.